triggers/runner/src/main/scala/com/daml/trigger/Runner.scala

```diff
@@ -177,13 +181,21 @@ object Converter {
       ("name", SText(id.entityName)))
   }
 
+  private def fromAnyContractId(triggerIds: TriggerIds, templateId: value.Identifier, contractId: String): SValue = {
```

Comment:
In principle that would be more consistent, yes. It's a bit noisy, though, because here we have a `value.Identifier`, but `AnyContractId` holds an `Identifier`. So, we'd be converting back and forth between the two identifier types.

Reply:
Ah I see, makes sense to keep it this way then. There are clearly too many things called `Identifier` here :slightly_smiling_face: